ssl/t1_trce.c

```diff
@@ -62,6 +62,8 @@ static ssl_trace_tbl ssl_version_tbl[] = {
     {TLS1_1_VERSION, "TLS 1.1"},
     {TLS1_2_VERSION, "TLS 1.2"},
     {TLS1_3_VERSION, "TLS 1.3"},
+    /* TODO(TLS1.3): Remove this line before release */
+    {TLS1_3_VERSION_DRAFT, TLS1_3_VERSION_DRAFT_TXT},
```

Comment:
Yes. The reason is so that I have only one place to update if we go from draft 18 to draft 19, i.e. in tls1.h:
````
/* TODO(TLS1.3) REMOVE ME: Version indicator for draft -18 */
# define TLS1_3_VERSION_DRAFT            0x7f12
# define TLS1_3_VERSION_DRAFT_TXT        "TLS 1.3 (draft 18)"
````
This is temporary anyway. By the time of release we shouldn't be using a draft version and it all gets deleted.